compiler/src/iree/compiler/InputConversion/MHLO/test/mhlo_to_mhlo_preprocessing.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: iree-opt --split-input-file --verify-diagnostics --iree-mhlo-to-mhlo-preprocessing %s | FileCheck %s
+// XFAIL: *
```

Comment:
do we have to disable all the tests? I thought it's a failure about scatter op?

Reply:
See Discord. I think this is just for testing... Please make sure this is removed before the integrate lands, and only the failing test is disabled (if it isnt root caused that is).

Reply:
I see, sorry for the spamming

Reply:
I updated the PR to only disable the failing tests. I also added a FIXME comment with the link to the issue: https://github.com/iree-org/iree/issues/10774